portality/scripts/application_status_report.py

```diff
@@ -59,27 +72,48 @@ def status_query(date_year, status):
 
     parser = argparse.ArgumentParser()
     parser.add_argument("-o", "--out", help="output file path")
-    parser.add_argument("-y", "--year", help="year to filter by")
+    parser.add_argument("-fy", "--from_year", help="from year to filter by")
```

Comment:
definitely time to add the `required=true` arg rather than checking manually below